### ISSUE TITLE
Upgrade samples to webpack 5 or other build tools

### ISSUE THREAD

**Issue by bossanovaorca** [AdobeDocs/uxp-photoshop-plugin-samples#90]
hello, I attempted to use the `ui-react-starter` template, but I discovered it was using webpack v4 which does not play nice with `sass`/`sass-loader`. when i attempted to upgrade webpack, the build time increased over 10x and often the builds would not even complete, they would just hang indefinitely. are there any plans to update these templates to support more recent dev dependency versions?

**Comment by yangdezuan** [comment]
yes,the webpack version is too low to run local by  v4, hope to upgraded them as soon as possilble.
